modules/gitops-configure-rollout-route-traffic-using-openshift-service-mesh.adoc

```diff
@@ -0,0 +1,491 @@
+      └──□ rollouts-demo-687d76d795-xrmrj  Pod         ✔ Running  9m50s  ready:1/1
+----
++
+--
+.Example with 60% of traffic directed to the stable version and 40% is directed to the canary version.
+[source,yaml]
+----
+route
+- destination:
+    host: rollouts-demo-stable
+  weight: 60 #<1>
+- destination:
+    host: rollouts-demo-canary
+  weight: 40 #<2>
+----
+<1> A value of `60` means that 60% of traffic is directed to the stable version.
+<2> A value of `40` means that 40% of traffic is directed to the canary version.
+--
+
+. Increase the traffic weight in the canary version to 100% and discard the traffic in the old stable version of the application by running the following command:
++
+--
+[source,terminal]
+----
+$ oc argo rollouts promote rollouts-demo -n <namespace> <1>
+----
+<1> Specify the namespace where the `Rollout` resource is defined.
+--
+
+.. Watch the progression of your rollout by running the following command:
++
+--
+[source,terminal]
+----
+$ oc argo rollouts get rollout rollouts-demo --watch -n <namespace> <1>
+----
+<1> Specify the namespace where the `Rollout` resource is defined.
+--
++
+In the given example, 60% is routed to the stable service and 40% of traffic is routed to the canary service. The deployment is then paused indefinitely until you request for promotion to the next level.
++
+.Example output
+[source,terminal]
+----
+Name:            rollouts-demo
+Namespace:       argo-rollouts
+Status:          ॥ Healthy
+Strategy:        Canary
+  Step:          8/8
+  SetWeight:     100
+  ActualWeight:  100
+Images:          argoproj/rollouts-demo:yellow (yellow)
+Replicas:
+  Desired:       5
+  Current:       5
+  Updated:       5
+  Ready:         5
+  Available:     5
+
+NAME                                       KIND        STATUS     AGE    INFO
+⟳ rollouts-demo                            Rollout     ✔ Healthy   19m  
+├──# revision:2                                                          
+│  └──⧉ rollouts-demo-6cf78c66c5           ReplicaSet  ✔ Healthy  99s    canary
+│     └──□ rollouts-demo-6cf78c66c5-zrgd4  Pod         ✔ Running  98s    ready:1/1
+└──# revision:1                                                          
+   └──⧉ rollouts-demo-687d76d795           ReplicaSet  ✔ Healthy  9m51s  stable
+      ├──□ rollouts-demo-687d76d795-75k57  Pod         ✔ Running  9m50s  ready:1/1
+      ├──□ rollouts-demo-687d76d795-jsxg8  Pod         ✔ Running  9m50s  ready:1/1
+      ├──□ rollouts-demo-687d76d795-rsgtv  Pod         ✔ Running  9m50s  ready:1/1
+      └──□ rollouts-demo-687d76d795-xrmrj  Pod         ✔ Running  9m50s  ready:1/1
+----
++
+--
+.Example with 0% of traffic directed to the canary version and 100% directed to the stable version.
+[source,yaml]
+----
+route
+- destination:
+    host: rollouts-demo-stable
+  weight: 100 #<1>
+----
+<1> A value of `100` means that 100% of traffic is directed to the stable version.
+--
```

Comment:
This entire block can be removed, as it is a duplicate from the previous step (and thus no longer relevant.)

Reply:
Fixed.